include/z64save.h

```diff
@@ -303,6 +303,18 @@ typedef enum {
     /* 0x06 */ HS_DAMPE_RACE
 } HighScores;
 
+// the score value for the fishing minigame also stores many flags.
+#define HS_FISH_LENGTH_CHILD 0x7F
+#define HS_FISH_LENGTH_ADULT 0x7F000000
```

Comment:
It would fit the OoT repo, but since z_fishing is so specialized and needs more passes anyway it's okay in general if not everything is perfectly polished (that stance also helps avoiding scope creeping)